src/main/java/com/infomaniak/lib/core/networking/HttpClient.kt

```diff
@@ -32,34 +32,35 @@ object HttpClient {
         this.tokenInterceptorListener = tokenInterceptorListener
     }
 
-    val okHttpClientNoTokenInterceptor: OkHttpClient by lazy { OkHttpClient.Builder().apply {
-        addCache()
-        addInterceptors()
-    }.build() }
+    val okHttpClientNoTokenInterceptor: OkHttpClient by lazy {
+        OkHttpClient.Builder().apply {
+            addCache()
+            addCommonInterceptors()
+        }.build()
+    }
 
     val okHttpClient: OkHttpClient by lazy {
         OkHttpClient.Builder().apply {
             addCache()
-            addInterceptors()
             addTokenInterceptor()
+            addCommonInterceptors()
         }.build()
     }
 
     val okHttpClientLongTimeoutNoTokenInterceptor: OkHttpClient by lazy {
         OkHttpClient.Builder()
             .apply {
                 addCache()
-                addInterceptors()
-                addCustomTimeout()
+                addCommonInterceptors()
```

Comment:
Is it normal that the custom timeout disappear here ? It is a Long timeout interceptor though